src/quo2/components/wallet/account_overview/view.cljs

```diff
@@ -0,0 +1,128 @@
+(ns quo2.components.wallet.account-overview.view
+  (:require [quo2.components.icon :as icons]
+            [quo2.components.markdown.text :as quo2]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [utils.i18n :as i18n]
+            [quo2.components.wallet.account-overview.style :as style]
+            [quo2.theme :as theme]))
+
+(defn- loading-state
+  [color]
+  [:<>
+   [rn/view (style/loading-bar-margin-bottom color 104 14)]
+   [rn/view {:style (style/loading-bar-margin-bottom color 152 20)}]
+   [rn/view {:style {:flex-direction :row}}
+    [rn/view {:style (style/loading-bar-margin-right color)}]
+    [rn/view {:style (style/loading-bar-margin-right color)}]
+    [rn/view {:style (style/loading-bar-margin-right-extra color 62 10 4)}]
+    [rn/view {:style (style/loading-bar-tiny color)}]]])
+
+(defn- account-overview-internal
+  [{:keys [account time-frame time-frame-string time-frame-to-string account-name current-value
+           percentage-change currency-change theme metrics loading?]}]
+
+  (let [time-frame-string       (case time-frame
+                                  :one-week     (i18n/label :t/one-week-int)
+                                  :one-month    (i18n/label :t/one-month-int)
+                                  :three-months (i18n/label :t/three-months-int)
+                                  :one-year     (i18n/label :t/one-year)
+                                  :all-time     (i18n/label :t/all-time)
+                                  :custom       time-frame-string)
+        up?                     (if (= metrics :positive)
```

Comment:
`if ... true false` could be removed here, leaving just `(= metrics :positive)` which returns true/false anyway